docs/reference/glossary.md

```diff
@@ -0,0 +1,243 @@
+
+### Cordoning
+
+Temporarily marking a node as unschedulable, preventing new pods from being assigned to it. In Devtron, you can cordon a node by going to Resource Browser → (choose a cluster) → Nodes → (click on a node) → Cordon (available in blue). [Read More...](https://docs.devtron.ai/usage/clusters#cordon-a-node)
+
+### CronJob
+
+A Kubernetes object that creates pods at specified intervals based on a cron schedule, commonly used for running periodic tasks. In Devtron, you can view a list of cronjobs by going to Resource Browser → (choose a cluster) → Workloads → CronJob. [Read More...](https://docs.devtron.ai/usage/applications/creating-application/deployment-template/job-and-cronjob#2.-cronjob)
+
+### Custom Charts
+
+Devtron offers a variety of ready-made Helm charts for common tasks and functions. If you have a specific need that isn't met by these preconfigured charts, super admins have the permission to upload their own custom charts. Once uploaded, these custom charts become accessible for use by all users on the Devtron platform. [Read More...](https://docs.devtron.ai/global-configurations/custom-charts)
+
+### DaemonSet
+
+A Kubernetes object that ensures a specific pod runs on all or certain nodes within a cluster, often used for tasks such as logging or monitoring. 
+
+In Devtron, you can view a list of DaemonSets by going to Resource Browser → (choose a cluster) → Workloads → DaemonSet.
+
+### Deployment Strategy
+
+A defined approach for deploying updates or changes to applications, which may include rolling updates, blue-green deployments, canary releases, and recreate strategy. 
+
+In Devtron, you can choose a deployment strategy by going to Applications (choose your app) → App Configuration (tab) → Workflow Editor → (edit deployment pipeline) → Deployment Strategy. [Read More...](https://docs.devtron.ai/usage/applications/creating-application/cd-pipeline#deployment-strategies)
+
+### Devtron Agent
+
+Your Kubernetes cluster gets mapped with Devtron when you save the cluster configurations. Now, the Devtron agent (rollout controller) must be installed from the chart store on the added cluster so that you can deploy your applications on that cluster. [Read More...](https://docs.devtron.ai/global-configurations/cluster-and-environments#installing-devtron-agent)
+
+### Devtron Apps
+
+The containerization of your source code followed by its deployment is totally configurable if you create a Devtron App. Custom apps deployed on Kubernetes cluster using the CI/CD feature of Devtron are called Devtron Apps. To create one, go to Applications → Create (button) → Custom App.
```

Comment:
This definition is good but looks a bit confusion?? Can we have something like,

Devtron Apps are the micro-services deployed with Kubernetes-native CI/CD with Devtron.